default.json

```diff
@@ -9,6 +9,8 @@
   "autodiscover": true,
   "autodiscoverFilter": ["balena-os/*"],
   "requireConfig": "optional",
+  "automergeType": "pr-comment",
```

Comment:
This is redundant as policy-bot will have auto-approved based on labels already. Unless we can test on an example repo that the PR comment comes later and not immediately, and then disable the policy-bot rule, this won't change the current behaviour.

Reply:
Yes, as I said above, policy-bot auto approval needs to be disabled for renovate to use pr-comment workflow, which may or may not work around this issue. Without policy-bot policy changes, this PR is a no-op.